sources/packages/api/src/services/education-program-offering/education-program-offering-import-csv.service.ts

```diff
@@ -66,7 +66,10 @@ export class EducationProgramOfferingImportCSVService {
       studyStartDate: csvModel.studyStartDate,
       studyEndDate: csvModel.studyEndDate,
       lacksStudyBreaks: csvModel.hasStudyBreaks == YesNoOptions.No,
-      studyBreaks: csvModel.studyBreaks,
+      studyBreaks:
+        csvModel.hasStudyBreaks == YesNoOptions.Yes
```

Comment:
`===`? is the object type different?

Reply:
It was a copy and paste from line 68 that was already wrong. I will change it.